### PR TITLE
Stringtable - Add & fix FR strings

### PR DESCRIPTION
**Pull request by Metalhearf**
#### Changes proposed in this pull request: 
- added french strings, in stringtable.xml
- fixed french semantic/meaning errors, in stringtable.xml
- changed currency from $ to €, in stringtable.xml _(only did it for in FR, but others european countries may need it)_

Have a nice day. ;)

### CODE REVIEW

**Comment by BoGuu** [comment]
Any french people that can check over this?

Also, for consistency, I'm not sure whether changing it to € is a wise idea. Not everything runs through the stringtable (and it's also just $ in vanilla by default). People can change this as they wish, on their own,

**Comment by Metalhearf** [comment]
Thanks for the review.
Yep seems better, I'll revert it back to $ in strings only then.

File: Altis_Life.Altis/stringtable.xml [hunk 1]
```diff
@@ -505,7 +505,7 @@
         <Key ID="STR_Admin_Amount">
             <Original>Put the amount you want to compensate:</Original>
             <Czech>Vložte částku, kterou chcete kompenzovat:</Czech>
-            <French>Mettez le montant que vous souhaitez compenser:</French>
+            <French>Mettez le montant que vous souhaitez compenser :</French>
```
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: Please delete the space before the `:`.

File: Altis_Life.Altis/stringtable.xml [hunk 2]
```diff
@@ -1041,7 +1041,7 @@
             <Italian>Il conto della gang non ha abbastanza denaro!</Italian>
             <Polish>Nie ma tego dużo na koncie bankowym gang!</Polish>
             <Russian></Russian>
-            <French>Vous n'avez pas autant dans votre gang!</French>
+            <French>Vous n'avez pas autant sur le compte de votre Gang !</French>
```
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: Please delete the space before the `!`.

File: Altis_Life.Altis/stringtable.xml [hunk 3]
```diff
@@ -1073,7 +1073,7 @@
             <Spanish>%1 ha transferido $%2 a tu cuenta.</Spanish>
             <Russian></Russian>
             <German></German>
-            <French></French>
+            <French>%1 vous a fait un virement de $%2</French>
```
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: End the sentence with a full stop, please.

File: Altis_Life.Altis/stringtable.xml (hunk 2 above)
Comment (Jawshy) [inline Altis_Life.Altis/stringtable.xml]: Would a lower case `g` in `gang` be more appropriate considering it is not a proper noun? Same goes for all of the other capitalised words. I'm not trying to be annoying, but we're trying to hold a high and consistency standard on the repository.

**Comment by Metalhearf** [comment]
Hi @Jawshy 
Thanks for the review. 

French punctuation is kinda different from the English one. We have to put a space before every ";", ":", "?" or "!"; and we don't have to put one before ",", "." "...". 
[Details here](https://www.quora.com/French-language-Is-the-rule-of-adding-a-space-between-the-end-of-a-sentence-and-an-exclamation-mark-question-mark-colon-used-a-lot)

There's still a lot of inconsistency between the strings in other languages, some of them do have a final "." and some don't. I'll try to clean it up for the FR strings. (I'll do it in two chunks of 5000 lines)

I agree with the rest of the reviews, I'll change it accordingly.

Thanks.

**Comment by Jawshy** [comment]
Thanks for clarifying French punctuation, @Metalhearf.

File: Altis_Life.Altis/stringtable.xml [hunk 4]
```diff
@@ -7098,7 +7098,7 @@
             <Spanish>Una carga explosiva ha sido puesta en la bóveda de la reserva federal, tienes hasta que se acabe el tiempo para desarmar el explosivo!</Spanish>
             <Russian></Russian>
             <German>Eine Sprengladung wurde am Safe angebracht. Du hast Zeit, bis die Uhr abläuft, um die Ladung zu entschärfen!</German>
-            <French>Une charge de dynamite a été placé sur le coffre de la réserve fédérale, Vous avez jusqu'au temps imparti pour désarmorcer la charge!</French>
+            <French>Une charge de dynamite a été placé sur le coffre de la réserve fédérale, Vous avez jusqu'au temps imparti pour désarmorcer la charge !</French>
```
Comment (setoy) [inline Altis_Life.Altis/stringtable.xml]: Wut?

Comment (setoy) [inline Altis_Life.Altis/stringtable.xml]: Disregard that. Didn't read the explanation above.

**Comment by Metalhearf** [comment]
Hi @Jawshy,
I'm happy with those changes ; went through all the file.
I think I'm done with commiting on this PR. 
See you !

**Comment by Jawshy** [comment]
LGTM

**Pull request merged**